docs/BUILD.md

```diff
@@ -1,65 +1,37 @@
-Build ipam-driver
-=================
-
-Prerequisite
-------------
-1. golang development environment is installed (https://golang.org/doc/install)
-
-
-Install Dependency
-------------------
-The driver primarily depends on libnetwork and the infoblox-go-client . They can be installed using the
-following commands:
+Building docker-ipam-plugin and docker-ipam-tools images
+========================================================
 
+Building Plugin Image
+---------------------
+To build the plugin image use the following command:
 ```
-go get github.com/docker/libnetwork  # libnetwork library (This also pulls down docker engine)
-go get github.com/docker/engine-api  # engine-api library (NOTE: run "make deps" to get dependencies)
-go get github.com/infobloxopen/infoblox-go-client  # Infoblox client
+make build-binary build-plugin create-plugin
```

Comment:
@ishantt , the whole point of `make` is to ensure that dependencies are built accordingly. The way it works now, one could be creating a plugin (using `make create-plugin`) that doesn't have the most up to date code.

To do what you want, you might want a special target for that. The targets for typical build should follow convention and build dependencies accordingly.